src/test/java/uk/gov/ons/ctp/response/notify/endpoint/StatusEndpointTest.java

```diff
@@ -73,7 +73,27 @@ public void setUp() throws Exception {
                 .build();
     }
 
-       /**
+    /**
+     * Scenario where null is returned from the database
+     *
+     * @throws Exception when getJson does
+     */
+    @Test
+    public void getStatusMessageNull() throws Exception {
```

Comment:
I wonder if NotFound would be a better description as it looks like that's the expectation?

Reply:
In that case, it's probably just a duplication of the test that follows and is not necessary to reimplement.